### DISCUSSION TITLE
ubuntu-20.04/Ubuntu 22.04.2 中无法运行附加了图标的程序

### DISCUSSION THREAD

**Discussion by haoishappy** [tw93/Pake#542] [Q&A]
![屏幕截图 2023-08-03 190339](https://github.com/tw93/Pake/assets/104285718/161439d8-4c46-4480-a393-bef43e06986e)
没有附加图标是可以正常运行Applmage的，但是附加icon后双击就运行不起来了，请问有知道该怎么解决吗
还有同样问题ProgramMusic安装好后，双击运行，一直转圈，无法正常开启
![屏幕截图 2023-08-03 191414](https://github.com/tw93/Pake/assets/104285718/ac8f3ed0-164b-4687-a521-64cc1963623b)

**Comment by Tlntin** [comment]
可以用命令行跑一下程序，看看报错信息。
